### DISCUSSION TITLE
NDI Video Source Issues in OBS 28.1.2

### DISCUSSION THREAD

**Discussion by priscillareiss** [DistroAV/DistroAV#814] [Q&A - Support]
Hi all,
I am using an Intel Mac and attempting to stream with my Elgato HD60S. I downloaded:
obs-ndi-4.10.0-Qt6-macOS.pkg 
NDI 4.5.1 macOS runtime
OBS 28.1.2 

However, NDI Video Source is not showing as NDI source or even as a Video Capture Device.
Any insight as to what may be the issue?

**Comment by mjbath** [comment]
I just found a guy on Fiverr.com (_process) that was able to help me out with this.  I had my music composition streaming setup working fine and then upgrading OBS to 28 messed everything up.  He was able to help out with that:  LINK REDACTED

**Comment by Trouffman** [comment]
You might not have isntalled the Runtime.

See the installation steps instructions.
